### ISSUE TITLE
argument of type 'module' is not iterable

### ISSUE THREAD

**Issue by patpatpatpatpat** [ierror/django-js-reverse#29]
Hey, any chance you could help me with this issue? Not sure if I'm missing something in the installation procedure. But I installed via pip, added `django_js_reverse` to installed apps, then added `  <script src="{% url 'js_reverse' %}" type="text/javascript"></script>` to my template. The file isn't loaded though. Accessing `localhost:8000/jsreverse` results to this error. 

```cmd
Environment:


Request Method: GET
Request URL: http://localhost:8000/jsreverse/

Django Version: 1.7
Python Version: 2.7.6
Installed Applications:
('django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'books',
 'django_js_reverse')
Installed Middleware:
('django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware')


Traceback:
File "/Users/pat/.virtualenvs/django1.7/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  111.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/Users/pat/.virtualenvs/django1.7/lib/python2.7/site-packages/django_js_reverse/views.py" in urls_js
  57.         response_body = minify(response_body, mangle=True, mangle_toplevel=False)
File "/Users/pat/.virtualenvs/django1.7/lib/python2.7/site-packages/slimit/minifier.py" in minify
  37.     parser = Parser()
File "/Users/pat/.virtualenvs/django1.7/lib/python2.7/site-packages/slimit/parser.py" in __init__
  56.         self.lexer.build(optimize=lex_optimize, lextab=lextab)
File "/Users/pat/.virtualenvs/django1.7/lib/python2.7/site-packages/slimit/lexer.py" in build
  102.         self.lexer = ply.lex.lex(object=self, **kwargs)
File "/Users/pat/.virtualenvs/django1.7/lib/python2.7/site-packages/ply/lex.py" in lex
  893.         if '.' not in lextab:

Exception Type: TypeError at /jsreverse/
Exception Value: argument of type 'module' is not iterable

```
My guess is it has to do with virtualenvs (I'm using virtualenvwrapper)?

Cheers!

**Comment by dahoste** [comment]
Just ran into exactly the same issue, on a fresh install of django-js-reverse in a venv.

Looks like a discrepancy between slimit, which uses the imported module lextab as the default 'lextab' argument when creating a Parser(), and the ply pkg, which assumes that the incoming 'lextab' argument is a string name for a module, rather than the module itself.

**Comment by dahoste** [comment]
This very recent change of ply broke the way slimit is using it:
https://github.com/dabeaz/ply/commit/daa1160e3b7e7bd94dc06da9e11ce9e188406fda#diff-d10ff62974a69de0294d7a52124c89e2

**Comment by dahoste** [comment]
'''JS_REVERSE_JS_MINIFY = False'''

Workaround for now: throw this in your settings.py.   That leaves slimit out of the loop, thus dodging the fact that slimit and ply are a broken combo right now.

**Comment by patpatpatpatpat** [comment]
@dahoste 
The `JS_REVERSE_JS_MINIFY = False` workaround works. Thanks!

**Comment by carljm** [comment]
You can also just `pip install ply==3.4` to use the 3.4 release instead of 3.6. It works just fine, and that way you still get minification.

**Comment by ierror** [comment]
Thank you all for debugging! I added ply==3.4 to the setup.py and released v0.5.1 to pypi

**Comment by ierror** [comment]
As of v0.6.1 slimit was replaced by rjsmin (included within django-js-reverse) so i could remove ply dependency
